commands/Gamba.py

```diff
@@ -0,0 +1,162 @@
+import supabase
+from typing import Optional
+import nextcord
+from nextcord.interactions import Interaction
+from nextcord.ui import *
+from nextcord.ext import commands
+
+class betInput(nextcord.ui.Modal):
+    def __init__(self, choice, supabase):
+        super().__init__(
+            "Your Bet")
+        
+        self.choice = choice
+        self.supabase = supabase
+
+        self.betAmount = nextcord.ui.TextInput(
+            label="Bet amount",min_length=1
+        )
+        
+        self.add_item(self.betAmount)
+
+    async def callback(self, interaction : nextcord.Interaction):
+            betValue = int(self.betAmount.value)
+            insertGambleData = {'id': interaction.user.id, self.choice: betValue}
+            existingUsersData = self.supabase.table('Users').select("coins").eq("id", interaction.user.id).execute().data[0]
+            if existingUsersData["coins"] < betValue:
+                await interaction.send("Your bet was not made. Insufficient funds.", ephemeral=True)
+                return
+            if self.supabase.table('customGamble').select('id').eq("owner", True).execute().data[0]['id'] == interaction.user.id:
+                await interaction.send("Your bet was not made, you created this gamble!", ephemeral=True)
+                return
+            newCoinBalance = existingUsersData["coins"] - betValue
+            updateUsersData = {"coins": newCoinBalance}
+            self.supabase.table('Users').update(updateUsersData).eq("id", interaction.user.id).execute().data
+            self.supabase.table('customGamble').upsert(insertGambleData).execute()
+            
+
+
+        
+
+class Dropdown(nextcord.ui.Select):
+    def __init__(self, supabase):
+        self.supabase = supabase
+        
+        self.selectOptions = [
+            nextcord.SelectOption(label="YES", description="Select if Yes won"),
+            nextcord.SelectOption(label="NO", description="Select if No won")
+        ]
+        super().__init__(placeholder="Cashout Options", min_values=1, max_values=1, options=self.selectOptions)
+    
+    
+    
+    
+
+    async def callback(self, interaction : Interaction):
+        fetchWinners = self.supabase.table('customGamble').select(f"id, {self.values[0]}").neq(self.values[0], 0).execute().data
+        totalWon = 0
+        for winner in fetchWinners:
+            userData = self.supabase.table('Users').select("id, coins").eq("id", winner['id']).execute().data[0]
+            updateData = {}
+            updateData['coins'] = int(userData['coins'])+ int((winner[self.values[0]]))*2
```

Comment:
Also the + missing whitespace before, and there doesn't need to be 2 sets of brackets around the second half. 

int((winner[self.values[0]]))*2 
could just be
int(winner[self.values[0]])*2